contrib/platform/src/com/sun/jna/platform/win32/ShellAPI.java

```diff
@@ -209,4 +211,853 @@ protected List getFieldOrder() {
+		 * flag (HOTKEYF_). For a list of modifier flags, see the description of
+		 * the <a href=
+		 * "https://msdn.microsoft.com/en-us/library/windows/desktop/ms646284(v=vs.85).aspx">
+		 * <strong xmlns="http://www.w3.org/1999/xhtml">WM_SETHOTKEY</strong>
+		 * </a> message. This member is ignored if <strong>fMask</strong> does
+		 * not include <strong>SEE_MASK_HOTKEY</strong>.
+		 * </p>
+		 */
+		public int dwHotKey;
+	
+		/**
+		 * This is actually a union:
+		 * 
+		 * <pre>
+		 * <code>union { HANDLE hIcon; HANDLE hMonitor; } DUMMYUNIONNAME;</code>
+		 * </pre>
+		 * 
+		 * <strong>DUMMYUNIONNAME</strong>
+		 * <dl>
+		 * <dt><strong>hIcon</strong></dt>
+		 * <dd>
+		 * <p>
+		 * <strong>Type: <strong>HANDLE</strong></strong>
+		 * </p>
+		 * </dd>
+		 * <dd>
+		 * <p>
+		 * A handle to the icon for the file type. This member is ignored if
+		 * <strong>fMask</strong> does not include
+		 * <strong>SEE_MASK_ICON</strong>. This value is used only in
+		 * Windows&nbsp;XP and earlier. It is ignored as of Windows&nbsp;Vista.
+		 * </p>
+		 * </dd>
+		 * <dt><strong>hMonitor</strong></dt>
+		 * <dd>
+		 * <p>
+		 * <strong>Type: <strong>HANDLE</strong></strong>
+		 * </p>
+		 * </dd>
+		 * <dd>
+		 * <p>
+		 * A handle to the monitor upon which the document is to be displayed.
+		 * This member is ignored if <strong>fMask</strong> does not include
+		 * <strong>SEE_MASK_HMONITOR</strong>.
+		 * </p>
+		 * </dd>
+		 * </dl>
+		 */
+		public HANDLE hMonitor;
+	
+		/**
+		 * <p>
+		 * Type: <strong>HANDLE</strong>
+		 * </p>
+		 * <p>
+		 * A handle to the newly started application. This member is set on
+		 * return and is always <strong>NULL</strong> unless
+		 * <strong>fMask</strong> is set to
+		 * <strong>SEE_MASK_NOCLOSEPROCESS</strong>. Even if
+		 * <strong>fMask</strong> is set to
+		 * <strong>SEE_MASK_NOCLOSEPROCESS</strong>, <strong>hProcess</strong>
+		 * will be <strong>NULL</strong> if no process was launched. For
+		 * example, if a document to be launched is a URL and an instance of
+		 * Internet Explorer is already running, it will display the document.
+		 * No new process is launched, and <strong>hProcess</strong> will be
+		 * <strong>NULL</strong>.
+		 * </p>
+		 * <div class="note"><strong>Note</strong>&nbsp;&nbsp;<a href=
+		 * "https://msdn.microsoft.com/en-us/library/windows/desktop/bb762154(v=vs.85).aspx">
+		 * <strong xmlns="http://www.w3.org/1999/xhtml">ShellExecuteEx</strong>
+		 * </a> does not always return an <strong>hProcess</strong>, even if a
+		 * process is launched as the result of the call. For example, an
+		 * <strong>hProcess</strong> does not return when you use
+		 * <strong>SEE_MASK_INVOKEIDLIST</strong> to invoke <a href=
+		 * "https://msdn.microsoft.com/en-us/library/windows/desktop/bb776095(v=vs.85).aspx">
+		 * <strong xmlns="http://www.w3.org/1999/xhtml">IContextMenu</strong>
+		 * </a>.</div>
+		 */
+		public HANDLE hProcess;
+	
+		@SuppressWarnings("rawtypes")
```

Comment:
Don't _@SuppressWarnings_ return a _List<String>_ instead (at some junction in time we will change the return type of the base class to it so less work for us when we do)